### PR TITLE
update makefiles, simulation and fpga files to support the new build method

### PR DESCRIPTION
**Pull request by codingUniv**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,9 +18,18 @@ populate-build-dir: $(VHDR) $(VSRC)
 	cp hardware/simulation/*.mk $(BUILD_DIR)/sim
 	mv $(BUILD_DIR)/sim/simulation.mk $(BUILD_DIR)/sim/Makefile
 	cp $(CORE_DIR)/hardware/simulation/*.expected $(BUILD_DIR)/sim
+ifneq ($(wildcard $(CORE_DIR)/hardware/simulation/*.mk),)
```
Comment (jjts) [inline Makefile:21]: remove this

File: Makefile [hunk 2]
```diff
@@ -18,9 +18,18 @@ populate-build-dir: $(VHDR) $(VSRC)
...
 	cp $(CORE_DIR)/hardware/simulation/*.mk $(BUILD_DIR)/sim
```
Comment (jjts) [inline Makefile:22]: do 
if [ '`ls $(CORE_DIR)/hardware/simulation/*.mk`' ]; then  cp $(CORE_DIR)/hardware/simulation/*.mk $(BUILD_DIR)/sim; fi

Comment (codingUniv) [reply Makefile]: Updated. Please refer to the new commit 8f0a087

Comment (codingUniv) [reply Makefile]: Update. Please refer to the new commit 7c90e48

**Pull request merged**